### PR TITLE
Alert conditions gif

### PR DESCRIPTION
**Pull request by nwhite222**
Replaced three images with dark mode versions and updated an out of date gif for creating alert conditions. NR-67824

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx [hunk 1]
```diff
@@ -16,9 +16,9 @@ redirects:
   - /docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries
 ---
 
-import queriesPolicyNewAlertCondition from 'images/queries-nrql_screenshot-crop_create-alert-condition.webp'
+import queriesPolicyNewAlertCondition from 'images/queries-nrql_screenshot_create-alert-condition.webp'
```
Comment (nbaenam) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:19]: This filename should be as follows: queries-nrql_screenshot-crop_create-alert-condition.webp. You need to indicate if is full or crop. You have [here](https://docs.newrelic.com/docs/style-guide/images/embed-images/#image-name-convention) the explanation.

**Review by nbaenam — COMMENTED** [review]
Hi Natasha! Thank you for the update! I've added a couple of comments, hope it helps! ;-) Let me know if you have questions.

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx [hunk 2]
```diff
@@ -16,9 +16,9 @@ redirects:
...
 
-import queriesMetricExplorer from 'images/queries-nrql_screenshot-crop_metrics-create-alert-condition.webp'
+import queriesMetricExplorer from 'images/queries-nrql-metrics-create-alert-condition.gif'
```
[the remark this answers is not in the document]
Comment (nwhite222) [reply src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:21]: Got it, thanks!

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx [hunk 3]
```diff
@@ -18,7 +18,7 @@ redirects:
 
 import queriesPolicyNewAlertCondition from 'images/queries-nrql_screenshot-crop_create-alert-condition.webp'
 
-import queriesMetricExplorer from 'images/queries-nrql_screenshot-crop_metrics-create-alert-condition.webp'
+import queriesMetricExplorerAnimation from 'images/queries-nrql_screenshot-crop_metrics-create-alert-condition.gif'
```
Comment (nbaenam) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:21]: I'm seeing that this is an animated gif. Just to make sure, have you asked in #doc_eng_help if we can use gif files? I understood that we have to use webp from now on, but maybe there's an exception with animated gifs.

**Review by nbaenam — APPROVED** [review]
Great job, Natasha! I've just left a comment, just to make sure that we can use gif files.

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx (hunk 3 above)
Comment (nwhite222) [reply src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx:21]: I just confirmed with Zach that .gifs are ok. Thank you!

**Pull request merged**